apps/src/templates/teacherDashboard/SectionRow.jsx

```diff
@@ -94,7 +94,8 @@ ConfirmDelete.propTypes = {
 
 const ProviderManagedSectionCode = ({provider}) => (
   <div data-tip={i18n.providerManagedSection({provider})}>
-    {i18n.none()}
+    {provider === 'clever' && i18n.loginTypeClever()}
+    {provider === 'google_classroom' && i18n.loginTypeGoogleClassroom()}
```

Comment:
Let's use the [`OAuthSectionTypes`](https://github.com/code-dot-org/code-dot-org/blob/c73c333a211267bcb567e7e07de03d71d7743389/apps/src/templates/teacherDashboard/shapes.jsx#L45) enum here.